ext_tables.sql

```diff
@@ -1,7 +1,7 @@
 --
--- Table structure for table 'tx_dlf_documents'
+-- Table structure for table 'tx_dlf_domain_model_document'
 --
-CREATE TABLE tx_dlf_documents (
+CREATE TABLE tx_dlf_domain_model_document (
```

Comment:
That's not really necessary. You can keep the "old" table names.

Reply:
That's good to know, becasue after reading few tutorials I have understood that the tables names should respond to the package in which the entity classes are placed. I was trying to find a way to inform the entity that table has actually different name.

How does Document entity knows that it accesses table tx_dlf_documents? 

In general I would be for unifing the table names to be only singular or only plural.

Reply:
This could be done by TypoScript mapping

e.g. like this in ext_typoscript_setup.txt:

```
config.tx_extbase.persistence.classes {
    Kitodo\Dlf\Domain\Model\Document {
        mapping {
            tableName = tx_dlf_documents
        }
    }
}
```

Ok, this has been dropped and replaced by a PHP class in TYPO3 10:

https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/10.0/Breaking-87623-ReplaceConfigpersistenceclassesTyposcriptConfiguration.html

(Good question, if we would need both, if we support TYPO3 9+10 in the  next release).

This could be a first step to start with the domain model while keeping all other code untouched.